android-sdk/Dockerfile

```diff
@@ -10,14 +10,19 @@ FROM ubuntu:20.04
 # ---------------------------------------------------------------------- #
 LABEL maintainer "thyrlian@gmail.com"
 
+# Disable Debian interactive by default, so automatic build won't hang on tzdata config
+ARG DEBIAN_FRONTEND=noninteractive
```

Comment:
This would not help.

The error still occurs:
```
E: Could not configure 'libc6:i386'.
E: Could not perform immediate configuration on 'libgcc-s1:i386'. Please see man 5 apt.conf under APT::Immediate-Configure for details. (2)
```

Reply:
`apt-get dist-upgrade -y` should work (to get rid of the above issue).  Don't know why it's not shown in the Travis CI logs.

Reply:
Nevermind.  I figured it out that there is some wrong config in my travis ci file, it's taking my master code and build instead of yours.  I'll fix that.

Reply:
Yep, I ended up adding `apt-get dist-upgrade -y` too, sorry I forgot to push the part of change back to this PR